modules/message-lane/src/lib.rs

```diff
@@ -525,6 +597,19 @@ impl<T: Config<I>, I: Instance> Module<T, I> {
 	}
 }
 
+/// Ensure that weights from `WeightInfo` implementation are looking correct.
+pub fn ensure_weights_are_correct<W: WeightInfoEx>() {
```

Comment:
So in the idea here that if we hit one of the `saturating` conditions in the weight calculations that we've done something wrong?

Reply:
Right. I was initially going to manually compare benchmarks results here, but `WeightInfoEx` is already doing the same thing (using `saturated_sub`).